core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -105,6 +109,16 @@ object DumpLogSegments {
         })
       }
     }
+
+    misMatchesForTimeIndexFilesMap.foreach {
+      case (fileName, listOfMismatches) => {
+        System.err.println("Mismatches in :" + fileName)
+        listOfMismatches.foreach(m => {
+          System.err.println("  Index timestamp: %d, log timestamp: %d".format(m._1, m._2))
```

Comment:
This is printing out the mismatches. The first timestamp is the timestamp in the index, and the second timestamp is the mismatch timestamp in the log. So the printed string seems correct?